src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -155,13 +158,14 @@ const JobHistory = _.flow(
     }
   })
 
-  const makeHeaderRenderer = (name, label, sortable = true) => {
+  const makeHeaderRenderer = (name, label, sortable = true, ariaLabel = label) => {
     return () => h(HeaderRenderer, {
       sort: sortable ? sort : undefined,
       name,
       label,
       onSort: setSort,
-      style: { fontWeight: 400 }
+      style: { fontWeight: 400 },
+      'aria-label': ariaLabel
```

Comment:
@breilly2 I added the ability to specify an ariaLabel so that "No. of Workspaces" reads out correctly.

Reply:
I also figured that for the first column, "click for details" was not helpful (and potentially confusing) in the spoken text.